tidb-server/main.go

```diff
@@ -685,3 +687,7 @@ func stringToList(repairString string) []string {
 		return r == ',' || r == ' ' || r == '"'
 	})
 }
+
+func setupSQLStatsProfiler() {
+	tracecpu.GlobalSQLStatsProfiler.Run()
```

Comment:
do we need a stop method? ( maybe we will meet goroutine leak in test? :)

Reply:
Currently no leak test for this.
I think no need to call `Stop`, it doesn't matter.
Mainly I don’t know when to call stop 😂

Reply:
When top sql enabled variable is switched to "off", then you need to stop the plrof I think.